mlos_bench/mlos_bench/config/schemas/config_schemas.py

```diff
@@ -98,5 +98,8 @@ def validate(self, config: dict) -> None:
         jsonschema.exceptions.ValidationError
         jsonschema.exceptions.SchemaError
         """
-        resolver: jsonschema.RefResolver = jsonschema.RefResolver.from_schema(self.schema, store=SCHEMA_STORE)
-        jsonschema.validate(instance=config, schema=self.schema, resolver=resolver)
+        # Allow skipping schema validation for tight dev cycle changes.
+        # Note: this may cause pytest to fail if it's expecting exceptions to be raised for invalid configs.
+        if environ.get('MLOS_BENCH_SKIP_SCHEMA_VALIDATION', 'false').lower() not in {'true', 'y', 'yes', 'on', '1'}:
```

Comment:
Potential future work improvement: fetching environment variables is somewhat expensive.  It might make sense to do this at script load time to cache the value and only check a boolean after that.

Reply:
I think we can do it later. Fetching an environment variable is still cheaper than validating a JSON